core/lib/dr_tools.h

```diff
@@ -341,6 +341,12 @@ typedef enum {
      * \note Windows only.
      */
     DR_MEMORY_DUMP_LDMP = 0x0001,
+    /**
+     * Memory dump in Executable and Linkable Format.
+     *
+     * \note Linux and X64 only.
```

Comment:
"X64 Linux only" would be more clear (as is, it could mean Linux is supported and X64 (Windows, Mac, etc.) is supported).

Reply:
I've updated the comment.